tests/test_authz_authorizer.py

```diff
@@ -1,35 +1,29 @@
 # coding=utf-8
+from copy import deepcopy
 from datetime import datetime, timedelta
+from unittest.mock import patch, MagicMock, call
 
 import pytest
 
-from lbz.authz import Authorizer, ALL, ALLOW, DENY, LIMITED_ALLOW
+from lbz.authz.authorizer import Authorizer, ALL, ALLOW, DENY, LIMITED_ALLOW
 from lbz.exceptions import PermissionDenied, SecurityRiskWarning, Unauthorized
 from tests import SAMPLE_PRIVATE_KEY, EXPECTED_TOKEN
 
 
 # pylint: disable=too-many-public-methods
-class TestAuthorizer:
-    def setup_method(self):
+class TestAuthorizerSetupClass:
+    @pytest.fixture(autouse=True)
```

Comment:
As I mentioned multiple times in #40: that is wrong usage of `pytest` fixtures, please get rid of that 😉